<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="FilenameEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="FilepathButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Filenumber">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PreviousButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilepathEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>641</width>
      <height>601</height>
     </rect>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>271</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_No1">
       <property name="title">
        <string>No.1</string>
       </property>
       <widget class="QPushButton" name="pushButton_find11">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>find</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_No1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Type List</string>
         </property>
        </item>
       </widget>
       <widget class="QGroupBox" name="groupBox_No2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>269</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>No.2</string>
        </property>
        <widget class="QPushButton" name="pushButton_find21">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_No2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Type List</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pushButton_find22">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find21">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find22">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_No3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>269</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>No.3</string>
        </property>
        <widget class="QPushButton" name="pushButton_find31">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_No3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Type List</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pushButton_find32">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find31">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find32">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_No4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>269</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>No.4</string>
        </property>
        <widget class="QPushButton" name="pushButton_find41">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_No4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Type List</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pushButton_find42">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find41">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find42">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_No5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>269</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>No.5</string>
        </property>
        <widget class="QPushButton" name="pushButton_find51">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_No5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Type List</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pushButton_find52">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find51">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find52">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_No6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>350</y>
          <width>269</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>No.6</string>
        </property>
        <widget class="QPushButton" name="pushButton_find61">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_No6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Type List</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pushButton_find62">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>find</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find61">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_find62">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_find12">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>find</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_find11">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>48</width>
          <height>21</height>
         </rect>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_find12">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>48</width>
          <height>21</height>
         </rect>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="type_list_btn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Type List setup</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>641</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Id_enable">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>54</y>
      <width>117</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID_Enable</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Id_disable">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>72</y>
      <width>117</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID_Disable</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>540</y>
      <width>94</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_x">
       <property name="text">
        <string>Position_ x   : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_y">
       <property name="text">
        <string>Position_ y   :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_width">
       <property name="text">
        <string>width             :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_height">
       <property name="text">
        <string>height            :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>540</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_x_value">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_y_value">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_width_value">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_height_value">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="FiletypeEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>png</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>66</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>FilenameEdit</zorder>
   <zorder>FilepathButton</zorder>
   <zorder>Filenumber</zorder>
   <zorder>OpenButton</zorder>
   <zorder>PreviousButton</zorder>
   <zorder>SaveButton</zorder>
   <zorder>FilepathEdit</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>type_list_btn</zorder>
   <zorder>widget</zorder>
   <zorder>radioButton_Id_enable</zorder>
   <zorder>radioButton_Id_disable</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>FiletypeEdit</zorder>
   <zorder>label</zorder>
   <zorder>NextButton</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDfdfdf">
   <property name="text">
    <string>dfdfdf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>PreviousButton</sender>
   <signal>clicked()</signal>
   <receiver>Filenumber</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PreviousButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NextButton</sender>
   <signal>clicked()</signal>
   <receiver>Filenumber</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NextButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
